dbt/dbt-run.py

```diff
@@ -28,7 +28,17 @@ def connection():
     with conn.cursor() as cur:
         cur.execute(f"""
             CREATE SCHEMA IF NOT EXISTS 
-            {os.getenv('POSTGRES_SCHEMA')}
+            {os.getenv('POSTGRES_SCHEMA')};
+
+            CREATE TABLE IF NOT EXISTS {os.getenv('POSTGRES_SCHEMA')}.{os.getenv('POSTGRES_TABLE')} (
```

Comment:
I think we should totally aim not _not_ create tables and such in code. I believe the whole allure of dbt is that you can have these versioned clean schema files, but here we're just inlining the table create in python? 
Is there a way we can use dbt to create these and keep the schema in its own file?

Reply:
Also, how is this connected to the helm chart?

Reply:
its not related to the the helm chart and was already merged to main separately.
dbt mainly creates tables by wrapping select from existing tables in create table statements, the assumption is that there's some source db that dbt is not managing.

but it is possible to just run raw sql, including ddl and yea i agree in this case probably makes more sense to do that instead of creating the table here. Also then could just have one "root" table instead of two.

related to https://github.com/medic/cht-pipeline/issues/84